### PR TITLE
Bump version number

### PR DESCRIPTION
**Pull request by golenkovm**
This is a fix for the issue #138

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2021051000;
+$plugin->version   = 2021081900;
```
Comment (dmitriim) [inline version.php:27]: should we add the same warning here?

Comment (golenkovm) [reply version.php:27]: I don't think so. 310 branch seems to be the main branch for Moodle 3.10 and onwards, so standard versioning is acpected here (when version number is bumped up to the current date)

Comment (dmitriim) [reply version.php:27]: well... it's the main branch now, but won't be in November when 4.0 is out. How will we remember to do it for this branch again? Should we document it somewhere else?

Comment (dmitriim) [reply version.php:27]: We will discuss the best approach the next internal dev  meeting as it should be applied to all plugins. So good to go as is for now.

**Review by dmitriim — APPROVED** [review]

**Pull request merged**